### DISCUSSION TITLE
Help please

### DISCUSSION THREAD

**Discussion by ghost** [gildor2/UEViewer#265] [General]
So one day today i was trying to export data and models from FNaF Security Breach and i was trying to export the MOD_ChicaPose01 Model But it came up with bad allocation size -358028800 bytes
i was looking on some discussions but that didint help me i was in the source code for umodel and then i found a file named astc_block_sizes2.cpp i dont know if this file is related to the bad allocation size but i dont know what im reading in it so if someone can help me ill be thankfull

**Comment by ghost** [comment]
oh and heres the call stack

appMalloc: size=-358028800 (total=14 Mbytes) <- FArray::Empty: 164039104 x 24 <- TArray::Serialize: 0/0 <- FStaticMeshLODModel4<< <- TArray::Serialize: 2/4 <- UStaticMesh4::Serialize <- LoadObject: StaticMesh4'MOD_ChicaPose01.MOD_ChicaPose01', pos=45994, ver=518/0 (unversioned), game=ue4.25 <- UObject::EndLoad <- LoadWholePackage: /Game/Models/Plushie/Chica/MOD_ChicaPose01.uasset <- ExportPackages <- CUmodelApp::ShowPackageUI <- Main: umodel_build=1585

**Comment by gildor2** [comment]
https://www.gildor.org/smf/index.php/topic,8037.0.html
The game is using UE4.25 Plus, you're setting 4.25

**Reply by ghost** [reply]
i tryed it but it did not help it still came up with the bad allocation size

**Comment by ghost** [comment]
Oh wait i think i got it working
